resubmit.py

```diff
@@ -251,9 +209,9 @@ def extendWorkflow(workflow, user, group, verbose=False, events=None, firstlumi=
     firstlumi = int(firstlumi)
 
     # Get info about the workflow to be cloned
-    helper = reqMgrClient.retrieveSchema(workflow)
-    schema = modifySchema(helper, workflow, user, group, None, events, firstlumi, None)
-    schema['OriginalRequestName'] = workflow
+    cache = reqMgrClient.getWorkflowInfo(url, workflow)
```

Comment:
nahh, this api
https://github.com/amaltaro/WmAgentScripts/blob/da9422cdeaa1af81cde15535b21d7c9a0093b777/reqMgrClient.py#L458
must yield that key/value.